dask/dataframe/io/parquet/arrow.py

```diff
@@ -313,6 +323,59 @@ def read_metadata(
 
         return (meta, stats, parts)
 
+    @staticmethod
+    def aggregate_row_groups(parts, stats, chunksize, split_row_groups=True, **kwargs):
```

Comment:
I was hoping that the aggregation could be agnostic to the engine, and could occur in core